content/en/data_jobs/databricks.md

```diff
@@ -43,7 +40,22 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
 1. In Databricks, click your display name (email address) in the upper right corner of the page.
 1. Select **Admin Settings** and click the **Compute** tab.
 1. In the **All purpose clusters** section, next to **Global init scripts**, click **Manage**.
-1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the init script.
+1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the following script, remembering to replace the placeholders with your parameter values. 
+
+   ```shell
+   #!/bin/bash
+
+   # Required parameters
+   export DD_API_KEY=<YOUR API KEY>
+   export DD_SITE=<YOUR DATADOG SITE>
+   export DATABRICKS_WORKSPACE=<YOUR WORKSPACE NAME> 
+
+   # Download and run the latest init script
+   bash -c "$(curl -L https://dd-data-jobs-monitoring-setup.s3.amazonaws.com/scripts/databricks/databricks_init_latest.sh)" || true
+   ```
+
+   The script above sets the required parameters and downloads the latest init script for Data Jobs Monitoring in Databricks. If you want to pin your script to a specific version, you can replace the file name in URL with `databricks_init_1.0.0.sh` to use the last stable version.
```

Comment:
```suggestion
   The script above sets the required parameters and downloads the latest init script for Data Jobs Monitoring in Databricks. If you want to pin your script to a specific version, you can replace the file name in the URL with `databricks_init_1.0.0.sh` to use the last stable version.
```

Reply:
Thank you @aliciascott. I made some changes based on your feedback. And at the same time, update our doc due to a slight different setup workflow, please review again. Thanks!